### ISSUE TITLE
Displayed values in Project page not tracking database values

### ISSUE THREAD

**Issue by j-sayers** [MotusDev/Motus-TO-DO#1293]
This was noticed previously with fee_id in #872 and now with fee_user_id. In both cases there is often a value populated/displayed on the website, but the corresponding field in the the projects table is NULL.

Using [project 536](https://motus.org/data/project/edit?id=536) as the  most recent example (created in August 2022), both fee_id and user_fee_id are NULL in the db, yet the corresponding dropdowns in the webpage (Institutional Agreement, and Administrative Contact) are both populated.

Where are these values coming from if not the projects table?

**Comment by j-sayers** [comment]
Yes I get that there are defaults, but the value displayed in the dropdown should reflect what's stored in the corresponding field in the database. But in some cases this isn't happening.

![image](https://user-images.githubusercontent.com/55001144/200350483-2a3c0330-c16d-443e-806f-1a47f5e5ba03.png)

![image](https://user-images.githubusercontent.com/55001144/200350640-1a02c870-1416-432f-8009-641e407f6406.png)

The Primary Contact dropdown and lead_user_id field agree with each other as they should (no NULLs in this field for any project).

But the other 3 dropdowns are displaying values that do not correspond to the database. 

If I go to the project and click save, the database will be updated. See below.

![image](https://user-images.githubusercontent.com/55001144/200351448-0132e81b-2368-4657-8772-0daeb8f8996c.png)

So it's a bit more clear what's happening now. The dropdowns are being populated with the defaults based on the criteria we have, but these values are not being written to the database as they should be.

**Comment by j-sayers** [comment]
@cjardine-bsc @denislepage can we update all projects with NULL as either `fee_id`, `fee_user_id`, `group_id` to whatever the default should be (1, the `lead_user_id` for the project, and 1).

New projects will still have NULL in the db for those populated fields (unless they hit save) but at least that will update all the existing missing values

**Comment by cjardine-bsc** [comment]
It is done.

**Comment by j-sayers** [comment]
thanks! Can you do the same for `fee_user_id` and `group_id`?

**Comment by cjardine-bsc** [comment]
I'll need to know the defaults.

**Comment by j-sayers** [comment]
fee_user_id should be the same as lead_user_id

group_id would be 1

**Comment by cjardine-bsc** [comment]
re: fee_user_id should be the same as lead_user_id

I can do that, but since they are in the same table, I think the fee_user_id would only apply if it's different than the lead_user_id wouldn't it?

group_id defaults updated.

**Comment by j-sayers** [comment]
Thank you!

> I can do that, but since they are in the same table, I think the fee_user_id would only apply if it's different than the lead_user_id wouldn't it?

I think it should still be populated with the default value instead of NULL, even though in most cases the values for the two fields will be the same.

**Comment by j-sayers** [comment]
@cjardine-bsc I see you are around and active.... If you agree on not having nulls can you update the remaining fee_user_id to match lead_user_id for those projects that still have null?

**Comment by cjardine-bsc** [comment]
For now lets leave them as NULL until I have a ponder about it. If you just want one column add this to your select statement. 

ISNULL(fee_user_id, lead_user_id) as user_id

**Comment by cjardine-bsc** [comment]
@cjardine-bsc  to flush out documentation of root issue for Brennen

**Comment by j-sayers** [comment]
Hurray for displayed values on the website corresponding to their values in the database!

We should also modify "open data" to "open complete data" to match the more descriptive term in the new collaboration.

At least that "do nothing" tooltip is only available to admins!

**Comment by bcreightonyoung** [comment]
@cjardine-bsc RE: gravy - I am getting a brief blurb upon hovering over the tooltip icons on /new. It sounds like you weren't. Regardless, they are still fairly unhelpful. Should I switch the tooltip layout on /new over to the paragraph-style or vise-versa (assuming we want both pages to have a consistent layout)?

**Comment by cjardine-bsc** [comment]
Paragraph style please. 

Thanks!

**Comment by bcreightonyoung** [comment]
Okay now a quick moral question: These two servlets are very similar already and I am now (literally) copying ~100 lines of code from one to the other. Should I stash the obvious refactor for another day?
